src/main/java/com/hotels/beeju/HiveMetaStoreJUnitRule.java

```diff
@@ -17,39 +17,18 @@
 
 import java.util.Map;
 
-import com.hotels.beeju.core.HiveMetaStoreCore;
 import org.apache.hadoop.hive.metastore.HiveMetaStoreClient;
 
+import com.hotels.beeju.core.HiveMetaStoreCore;
+
 /**
  * A JUnit {@link Rule} that creates a Hive Metastore backed by an HSQLDB in-memory database.
```

Comment:
This showed in red for me in IntelliJ. But after adding an import for `org.junit.Rule`, it looked fine. And there are 2 other classes that have javadocs rules for unimported linked classes. It looks like these are unresolved references inside the Javadoc. Do we care about these or not?